Sources/Commands/SwiftPackageTool.swift

```diff
@@ -863,6 +865,138 @@ extension SwiftPackageTool {
             swiftTool.outputStream.flush()
         }
     }
+    
+    // Experimental command to invoke user command plugins. This will probably change so that command that is not
+    // recognized as a built-in command will cause `swift-package` to search for plugin commands, instead of using
+    // a separate `plugin` subcommand for this.
+    struct PluginCommand: SwiftCommand {
```

Comment:
Note again: using a separate `plugin` subcommand is temporary until I figure out how to implement this as a fallback from the general command handler.